### PR TITLE
Tema7

### PR DESCRIPTION
**Pull request by SanduBogdan95**

### CODE REVIEW

File: src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class CURAHealthcare {
+    static ChromeDriver driver = null;
+
+    @BeforeMethod
```
Comment (95teo) [inline src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java:15]: Can you please move @BeforeMethod and @AfterMethod to a separated abstract TestBase class and your test classes to extend that TessBase class? You have an example in the demo for chapter 7 from Timea. Thanks!

File: src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        //open home page
+        driver = BrowserManager.createDriverAndGetPage();
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        driver.manage().window().maximize();
+        System.out.println("Am deschis CURA Healthcare Service page");
+    }
+
+    @Test
+    public void login() {
+        // go to login page
```
Comment (95teo) [inline src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java:26]: This method and invalidlogin() method can be merged into one method if you use @DataProvider. Try this approach and let me know if you have any issues. Thanks!

File: src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class CURAHealthcare extends TestBase {
```
Comment (95teo) [inline src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java:15]: You're not using any Assert. Consider adding on test cases.

File: src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
```
Comment (95teo) [inline src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java:16]: You're missing bullet 3 and 5 from ex 2.

File: src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public void completeDetailsForAppointment(){
+        driver.findElement(By.id("txt-username")).sendKeys("John Doe");
+        driver.findElement(By.id("txt-password")).sendKeys("ThisIsNotAPassword");
+        driver.findElement(By.id("btn-login")).click();
+    }
+
+    @Test(dataProvider = "data-set")
+    public void makeAppointment(String type, String username, String password){
```
Comment (95teo) [inline src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java:50]: Please double check the requirement. In order to be on appointment page you need to be logged in -> Se vor testa diferite scenarii pozitive și negative de rezervare (măcar 1 scenariu pozitiv și un scenariu negativ) Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by ioanaandroflorea — APPROVED** [review]

**Review by 95teo — APPROVED** [review]

**Pull request merged**